index.html

```diff
@@ -1,12 +1,113 @@
 <html>
 <head>
-    <title>Concept Opdrachten</title>
+    <link rel="stylesheet" href="styles.css">
+    <title>Index Pagina Opdrachten</title>
 </head>
 <body>
-<header></header>
+<header class="header">
+    <nav>
+        <ul class="nav-list">
+            <li class="nav-item"><a href="#casestudies"> Case Studies</a></li>
+            <li class="nav-item"><a href="#services"> Services</a></li>
+            <li class="nav-item"><a href="#contactus"> Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="first-row">
+    <h1>Your website is an investment in the success of your business.</h1>
+        <h4>We are here to help you succeed</h4>
+        <div class=""><input type="submit" name="work-together" VALUE="LET'S WORK TOGETHER" class="button-work-together"></div>
+       <div class="squiggle-stripe"> <img src="/assets/squiggle_line_stripe.svg" width="100" height="100"></div>
```

Comment:
Voor alle afbeeldingen geldt dat ik de `/` weg moeten halen, omdat ik anders de afbeeldingen niet kan zien
```suggestion
       <div class="squiggle-stripe"> <img src="assets/squiggle_line_stripe.svg" width="100" height="100"></div>
```

Reply:
Bedankt !! Ik ga deze veranderen.